### DISCUSSION TITLE
我用 Ant Design Pro 写了一个通用的管理后台，大家给提提意见

### DISCUSSION THREAD

**Discussion by tangtanglove** [ant-design/ant-design-pro#7555] [Ideas]
![微信截图_20201104194552](https://user-images.githubusercontent.com/60868863/98108157-82217f00-1ed6-11eb-8f67-bfdcbf25149e.png)

## 演示站点

网址：[http://admin.quarkcms.com/admin/index](http://admin.quarkcms.com/admin/index)

用户名：administrator 密码：123456

## 官方网站
<http://www.quarkcms.com/>
github CMS仓储:[quark-cms](https://github.com/quarkcms/quark-cms)
github 后端仓储:[quark-admin](https://github.com/quarkcms/quark-admin)
github 前端仓储:[quark-ui](https://github.com/quarkcms/quark-ui)

**Comment by tangtanglove** [comment]
@afc163 大佬 @chenshuai2144 帅哥帮兄弟顶顶吧，废了很多精力和心血。

**Comment by aixgeek** [comment]
后端建议用serverless方案

quarkcms <notifications@github.com> 于2020年11月5日周四 上午9:30写道：

> @afc163 <https://github.com/afc163> 大佬 @chenshuai2144
> <https://github.com/chenshuai2144> 帅哥帮兄弟顶顶吧，废了很多精力和心血。
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ant-design/ant-design-pro/issues/7555#issuecomment-722067945>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJ7TM4UY42WZBEOTE2T44TSOH54BANCNFSM4TJ6M34A>
> .
>


--

**Comment by tangtanglove** [comment]
公司后端暂时还是用PHP的技术栈，换方案成本太高

**Comment by chenshuai2144** [comment]
给你点赞，可以写个文章推广推广

**Comment by tangtanglove** [comment]
嗯嗯，正尝试推广呢；但是现在有点儿困难，大多数写后台的对于Vue还能懂一点，React都不太懂；写前台的又对于后端不太熟悉！而对于两方面都懂的人又看不上我写的项目。所以头疼。

**Reply by jefferyqjy** [reply]
我是java+antd pro（react）有一阵子了， 你这个后端是用的什么啊~？

**Comment by chenshuai2144** [comment]
可以试试后端也用node，或者打造搞个卖点出来

**Comment by tangtanglove** [comment]
嗯，后端换node技术栈，对于团队的学习成本太高；现在团队是PHP的技术栈，万事开头难，慢慢来吧。

**Comment by jipfqf** [comment]
大佬，有时间搞搞富文本吗

**Comment by tangtanglove** [comment]
> 大佬，有时间搞搞富文本吗

富文本也是支持的

**Comment by jipfqf** [comment]
> > 大佬，有时间搞搞富文本吗
> 
> 富文本也是支持的

demo上没看到

**Comment by tangtanglove** [comment]
嗯嗯，演示站确实没有，功能是有的，你可以试一下

**Comment by zchhaenngg** [comment]
一个字  牛
